services/spotify_service.py

```diff
@@ -1,45 +1,61 @@
 
-    def recommend(self, access_token, user_id, sentiment_value):
-        spotify_recommendation = self.get_spotify_recommendation(access_token, sentiment_value)
 
+    def recommend(self):
+        spotify_recommendation = self.get_spotify_recommendation()
         if spotify_recommendation.status_code == 200:
             body = spotify_recommendation.json()
-            return { 'message': {
-                    'artist': body['tracks'][0]['artists'][0]["name"],
-                    'track_id': body['tracks'][0]['id'],
-                    'track_name': body['tracks'][0]['name'],
-                    'track_uri': body['tracks'][0]['uri']
-                    }, 'status_code': 200 }
+            return self._recommended_tracks(body)
         elif spotify_recommendation.status_code == 401:
-            request = requests.get(f'https://guten-server.herokuapp.com/api/v1/access_token/{user_id}')
-
-            if request.status_code == 200:
-                access_token = request.json()['access_token']
-                return self.recommend(access_token, user_id, sentiment_value)
-            else:
-                return { 'message': f"User:{user_id} does not exist", 'status_code': 404}
+            return self._expired_token()
         else:
-            return { 'message': "Bad request", 'status_code': 400 }
+            return self._bad_request()
+
 
-    def song_params(self, sentiment_value):
+    def song_params(self):
         params = {
-            'valence': sentiment_value,
             'seed_genres': 'classical',
-            'limit': 1,
-        }
-        if sentiment_value == 1:
-            params['mode'] = 1
-        elif sentiment_value == 0:
-            params['mode'] = 0
-
+            'limit': 10 }
+        if self.sentiment == 'Positive':
+            params['mode']    = 1
+            params['valence'] = 1
+        elif self.sentiment == 'Negative':
+            params['mode']    = 0
+            params['valence'] = 0
+        else:
+            params['valence'] = 0.5
         return params
 
 
-    def get_spotify_recommendation(self, access_token, sentiment_value):
-        params = self.song_params(sentiment_value)
-        headers = {'Authorization': f'Bearer {access_token}'}
-        request = requests.get('https://api.spotify.com/v1/recommendations', headers=headers, params=params)
-        return request
+    def get_spotify_recommendation(self):
+        params = self.song_params()
+        headers = {'Authorization': f'Bearer {self.access_token}'}
+        response = requests.get(
+            'https://api.spotify.com/v1/recommendations',
+            headers=headers,
+            params=params)
+        return response
+
+
+    def _bad_request(self):
+        return { 'message': 'Bad request', 'status_code': 400 }
+    
+
+    def _expired_token(self):
+        return { 'message': 'The access token expired', 'status_code': 401}
+    
+
+    def _recommended_tracks(self, body):
+        track_list = []
```

Comment:
Thanks for refactoring this method to its own private method. I like this better than when I originally wrote the method in the recommendation function.